tools/doc/type-parser.js

```diff
@@ -105,6 +107,7 @@ const customTypesMap = {
   'Timeout': 'timers.html#timers_class_timeout',
   'Timer': 'timers.html#timers_timers',
 
+  'tls.SecureContext': '#tls_tls_createsecurecontext_options',
```

Comment:
Is it intentional that it does not start with `tls.html#`? The other tls entries have that as a start.

Reply:
No, it's a typo. Thank you! I will fix in a fixup commit.